doc/datavolumes.md

```diff
@@ -0,0 +1,136 @@
+# Data Volumes
+
+## Introduction
+Data Volumes(DV) are an abstraction on top of Persistent Volume Claims(PVC) and the Containerized Data Importer(CDI). The DV will monitor and orchestrate the upload/import of the data into the PVC. Once the process is completed, the DV will be in a consistent state that allow consumers to make certain assumptions about the DV in order to progress their own orchestration.
+
+Why is this an improvement over simply looking at the state annotation created and managed by CDI? Data Volumes provide a versioned API that other project like Kubevirt](https://github.com/kubevirt/kubevirt) can integrate with. This way those project can rely on an API staying the same for a particular version and have guarantees about what that API will look like. Any changes to the API will result in a new version of the API.
+
+### Status phases
+The following statuses are possible.
+* 'Blank': No status available.
+* Pending: The operation is pending, but has not been scheduled yet.
+* PVCBound: The PVC associated with the operation has been bound.
+* Import/Clone/UploadScheduled: The operation (import/clone/upload) has been scheduled.
+* Import/Clone/UploadInProgress: The operation (import/clone/upload) is in progress.
+* Succeeded: The operation has succeeded.
+* Failed: The operation has failed.
+* Unknown: Unknown status.
+
+## HTTP/S3 source
+Data Volumes are an abstraction on top of the annotations one can put on PVCs to trigger CDI. As such DVs have the notion of a 'source' that allows one to specify the source of the data. To import data from an external source, the source has to be either 'http' or 'S3'. If your source requires authentication, you can also pass in a secretRef to a secret containing the authentication information.
+
+```yaml
+apiVersion: cdi.kubevirt.io/v1alpha1
+kind: DataVolume
+metadata:
+  name: "example-import-dv"
+spec:
+  source:
+      http:
+         url: "https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img" # Or S3
+         secretRef: "" # Optional
```

Comment:
What is the expected format of the secret you reference?  Does that secret need to contain specific named fields?

Reply:
Yeah, would be really good if we had an example for this

Reply:
Added some links to an example secret.